src/state/page-config/actions.js

```diff
@@ -1,24 +1,24 @@
 import { getParams } from 'frontend-common-components';
-import { SET_CONTENT_TOOLBAR, SET_SEARCH_FILTER, CHANGE_VIEW_LIST } from 'state/page-config/types';
+import { SET_CONTENT_TOOLBAR, SET_SEARCH_FILTER, CHANGE_VIEW_LIST, EXPAND_CONTENT_TOOLBAR } from 'state/page-config/types';
 
 import { addErrors } from 'state/errors/actions';
 import { setSelectedPageModel } from 'state/page-models/actions';
 import { fetchPage } from 'api/pages';
 import { getPageModel } from 'api/pageModels';
 
-export const setContentToolbar = content => ({
+export const setContentToolbar = () => ({
   type: SET_CONTENT_TOOLBAR,
-  payload: {
-    content,
-  },
+});
+
+export const expandContentToolbar = () => ({
+  type: EXPAND_CONTENT_TOOLBAR,
```

Comment:
You are using this action to toggle the expanded state (if true, set to false, else set to true), and not only to expand.

Please name the action (and the type) TOGGLE instead of EXPAND